custom_components/mikrotik_router/update.py

```diff
@@ -150,3 +155,8 @@ async def async_install(self, version: str, backup: bool, **kwargs: Any) -> None
         """Install an update."""
         self._ctrl.execute("/system/routerboard", "upgrade", None, None)
         self._ctrl.execute("/system", "reboot", None, None)
+
+    @property
+    def title(self) -> str:
+        """Title of the update entity."""
+        return "Mikrotik RouterBoard"
```

Comment:
I think this should be "RouterBOARD firmware"
Also, should be implemented through entity descriptor, instead of hardcoded

Reply:
See above

Reply:
Thats fair, still should be implemented wia entity descriptor

Reply:
Done! I hope it's ok for you, and please, feel free to change whatever you see fit :-)